contrib/inventory/cobbler.py

```diff
@@ -190,7 +190,7 @@ def update_cache(self):
                         if this_dns_name is not None and this_dns_name is not "":
                             dns_name = this_dns_name
 
-            if dns_name == '':
+            if dns_name == '' or dns_name is None:
```

Comment:
I agree that in this case, `if not dns_name` would probably work. I can't think of a case where the Cobbler API will return any other [Python Falsey values](https://docs.python.org/3/library/stdtypes.html#truth-value-testing) for the value of the DNS name.

I tend to prefer being explicit about my comparisons. It avoids bugs caused unexpected inputs. 

If there is a coding standard or some "Pythonic" rule I'm breaking, I'd be happy to change it.